test/ctr.bats

```diff
@@ -32,125 +34,66 @@ function wait_until_exit() {
 
 @test "ctr not found correct error message" {
 	start_crio
-	run crictl inspect "container_not_exist"
-	echo "$output"
-	[ "$status" -eq 1 ]
-
-	stop_crio
+	! crictl inspect "container_not_exist"
 }
 
 @test "ctr termination reason Completed" {
 	start_crio
-	run crictl runp "$TESTDATA"/sandbox_config.json
-	echo "$output"
-	[ "$status" -eq 0 ]
-	pod_id="$output"
-	run crictl create "$pod_id" "$TESTDATA"/container_config.json "$TESTDATA"/sandbox_config.json
-	echo "$output"
-	[ "$status" -eq 0 ]
-	ctr_id="$output"
-	run crictl start "$ctr_id"
-	echo "$output"
-	[ "$status" -eq 0 ]
+	pod_id=$(crictl runp "$TESTDATA"/sandbox_config.json)
+	ctr_id=$(crictl create "$pod_id" "$TESTDATA"/container_config.json "$TESTDATA"/sandbox_config.json)
+	crictl start "$ctr_id"
 	wait_until_exit "$ctr_id"
-	[ "$status" -eq 0 ]
-	run crictl inspect --output yaml "$ctr_id"
-	echo "$output"
-	[ "$status" -eq 0 ]
+
+	output=$(crictl inspect --output yaml "$ctr_id")
 	[[ "$output" == *"reason: Completed"* ]]
 }
 
 @test "ctr termination reason Error" {
 	start_crio
-	run crictl runp "$TESTDATA"/sandbox_config.json
-	echo "$output"
-	[ "$status" -eq 0 ]
-	pod_id="$output"
-	errorconfig=$(cat "$TESTDATA"/container_config.json | python -c 'import json,sys;obj=json.load(sys.stdin);obj["command"] = ["false"]; json.dump(obj, sys.stdout)')
-	echo "$errorconfig" > "$TESTDIR"/container_config_error.json
-	run crictl create "$pod_id" "$TESTDIR"/container_config_error.json "$TESTDATA"/sandbox_config.json
-	echo "$output"
-	[ "$status" -eq 0 ]
-	ctr_id="$output"
-	run crictl start "$ctr_id"
-	echo "$output"
-	[ "$status" -eq 0 ]
+	pod_id=$(crictl runp "$TESTDATA"/sandbox_config.json)
+
+	python -c 'import json,sys;obj=json.load(sys.stdin);obj["command"] = ["false"]; json.dump(obj, sys.stdout)' \
```

Comment:
why not edit_json here?

Reply:
It was not there yet at the time I wrote this.

I am going to have a separate PR attacking this python and sed (and trying to reduce the number of testdata files).